Drop the paths from LIBUCONTEXT_HEADERS when installing, so that `libucontext.h`
will find `bits.h`.
--- a/Makefile	2024-10-07 03:37:08.000000000 +0200
+++ b/Makefile	2025-03-21 21:16:16.761054357 +0100
@@ -215,8 +215,8 @@ install: all
 	install -D -m664 ${LIBUCONTEXT_STATIC_NAME} ${DESTDIR}${LIBUCONTEXT_STATIC_PATH}
 	ln -sf ${LIBUCONTEXT_SONAME} ${DESTDIR}${shared_libdir}/${LIBUCONTEXT_NAME}
 	for i in ${LIBUCONTEXT_HEADERS}; do \
-		destfn=$$(echo $$i | sed s:include/::g); \
-		install -D -m644 $$i ${DESTDIR}${includedir}/$$destfn; \
+		destfn=$$(basename $$i); \
+		install -D -m644 $$i ${DESTDIR}${includedir}/$(basename $(LIBUCONTEXT_NAME))/$$destfn; \
 	done
 	install -D -m644 ${LIBUCONTEXT_PC} ${DESTDIR}${pkgconfigdir}/${LIBUCONTEXT_PC}
 	if [ -n "${LIBUCONTEXT_POSIX_NAME}" ]; then \
